mint-001.md

```diff
@@ -0,0 +1,182 @@
+# mint-001
+
+## 3 Key Degrading Multisig
+
+## Motivation
+
+The 2-of-3 multisig serves as a standard in multisignature solutions, offering a well-balanced combination of redundancy, security, flexibility, and practicality for securing Bitcoin. Loss of a single key does not spell disaster, and key distribution enhances security. Miniscript's time feature enables further disaster recovery. If two keys are lost, a timelock can convert the wallet to a 1-of-3 multisig. The proposal outlines four conditions through Miniscript's `thresh()`:
+
+### 2 Layer Conditions to be satisfied using:
+
+#### Base expressions (type B).
+
+##### timelock: **relative** or **absolute** [^timelock] [^either]
+
+- **older(**int**)** [^older]
+
+- **after(**int**)** [^after]
+
+#### Key expressions[^k_type] (type K).
+
+- **pk(**$Key_1$**)**
+
+- **pk(**$Key_2$**)**
+
+- **pk(**$Key_3$**)**
+
+Initially, the wallet requires 2-of-3 keys, functioning as a traditional 2-of-3 multisig. After the timelock expires, only one key is needed to spend the funds.
+
+### More on Timelock Values
+
+- Descriptor template security is independent of relative or absolute timelock values; the impact is on timelock duration only. Reference transactions on testnet use short-duration timelocks; however, practical applications will vary.
```

Comment:
In ab8f15bdfc9751d373f29da38a491d8948c4df29:

I find this "security is independent of" claim confusing. The value of the timelocks determines what the policy is and that feels like a "security property" to me.

Maybe you mean to say something like: there are no valid timelock values that change the security or correctness properties beyond those implied by the timelocks themselves.

And maybe we can drop the word "valid" since invalid timelock values really shouldn't be accepted by Miniscript..

Reply:
Attempted to incorporate your suggested language:

https://github.com/Blockstream/miniscript-templates/pull/11/files#diff-20e8f9b39b392093607d290164c4379e5e6de17e221756cc96b4149387ba5d61R29